javaHapiValidatorLambda/serverless.yml

```diff
@@ -0,0 +1,35 @@
+#
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+service: fhir-service-validator
+
+provider:
```

Comment:
Could you add logRetention and StackTag? https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/serverless.yaml#L28-L30